core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocModel.java

```diff
@@ -506,6 +516,8 @@ ElementType findOrCreateElementType(Class<?> clazz) throws ReflectiveOperationEx
 				log.trace(String.format("Class [%s] is a Java interface, going to create all member FrankElement", clazz.getName()));
 			}
 			List<SpringBean> springBeans = Utils.getSpringBeans(clazz.getName());
+			// We sort here to make the order deterministic.
+			Collections.sort(springBeans);
```

Comment:
Waarom is dit nu nodig en eerst niet?

Reply:
Dit had ik al eerder moeten doen, maar nu valt het me op.

Reply:
Ok, het heeft ermee te maken dat we voor de XSD met ElementRole werken. Alleen voor de JSON browsen we direct ElementType.